packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -33,6 +33,24 @@ class DevInstall extends NodePackageInstallTask {
   }
 }
 
+/**
+ * Retrive the package manager from the angular.json
+ *
+ * @param tree
+ * @returns package manager as string or undefined
+ */
+function getPackageManagerFromTree(tree: Tree) {
+  let packageManager;
+  const angularJsonString = tree.read('/angular.json')?.toString();
+  if (angularJsonString) {
+    const angularJsonObj = JSON.parse(angularJsonString);
+    if (angularJsonObj?.cli?.packageManager) {
```

Comment:
we could do the same check on `nx.json` or do we want to handle Nx support later ?

Reply:
from what I have seen, if we run in nx context, we have an `angular.json` in the tree built from nx.json and project.json.
I have checked how the package manager is declared in nx.json and it is still under cli.package.manager same as in angular.json. So that's why this code should be compatible with nx too.
@kpanot any thoughts ?

Reply:
finally done in https://github.com/AmadeusITGroup/otter/pull/473